### ISSUE TITLE
Graphical issues with Navigation Bar

### ISSUE THREAD

**Issue by VitaTaf** [VitaTaf/usc#2]
For example, when launching Sleep as Android, the Sleep "emoji" on the home button is blocked by the opaque navigation bar. (This bug is present when using the Nexus Launcher)
Occasionally, observed in Google Duo and Play Books, immersive full-screen mode causes a black bar where the navigation bar used to be before hiding. However, this may be a separate issue than the one stated above.

origin: undetermined

possible related logcat:
`W/art     ( 6754): Before Android 4.1, method android.graphics.PorterDuffColorFilter android.support.graphics.drawable.VectorDrawableCompat.updateTintFilter(android.graphics.PorterDuffColorFilter, android.content.res.ColorStateList, android.graphics.PorterDuff$Mode) would have incorrectly overridden the package-private method in android.graphics.drawable.Drawable
I/SleepAsAndroid( 6754): [19:46:37.075, main]: StatusBar: Cover view android.view.View{eb6cc5d G.ED..... ......I. 0,0-0,0 #7f1001cf app:id/tint_cover_view}`
